extension/src/editor/containers/common/commonTemplate/TemplateHeader.tsx

```diff
@@ -167,3 +211,16 @@ const StyledButton = styled(Button)(({ theme }) => ({
     borderRadius: "0",
   },
 }));
+
+function generateDuplicateTemplateName(ori: string) {
+  return `${ori} (copy-${new Date().getTime()})`;
```

Comment:
It looks like non-human readable number. Is it no problem? I thought this number confuses user.